tests/test_color_palette.py

```
@@ -1,251 +1,126 @@
-def get_color_palette():
-    """Get the color palette from tabs/_common.py"""
+def get_styling_data():
+    """Get color palette data from the actual source function."""
     try:
-        from tabs._common import get_color_palette as fetch_colors
+        # ใน tabs/_common.py มีฟังก์ชัน get_color_palette
+        from tabs._common import get_color_palette
 
-        return fetch_colors()
+        palette = get_color_palette()
+        return {"colors": palette, "fetcher": get_color_palette}
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)